### DISCUSSION TITLE
How to render transparent textures

### DISCUSSION THREAD

**Discussion by Haidiye00** [google/filament#7008] [Q&A]
![F8Wyrj8o8w](https://github.com/google/filament/assets/132528822/2780efea-db99-48fd-9438-19b35835248f)
This is the effect I rendered based on Filament's texturequad example, but my image is not surrounded by black, but transparent. How to set and solve it. The shader is blakedTexutre.mat

**Comment by bejado** [comment]
You need to set a [blending mode](https://google.github.io/filament/Materials.html#materialdefinitions/materialblock/blendingandtransparency:blending) in your material and ensure the `.a` alpha channel of the color output is set correctly in the fragment shader.

**Comment by Haidiye00** [comment]
ok,i solve it ,thanks
